### ISSUE TITLE
The dropout and batchnormalization

### ISSUE THREAD

**Issue by zeyayin** [kentsommer/tensorflow-posenet#8]
Hello
It seems you write the function for dropout and batchnormalization。
But I did not find where you implement them in poseNet。

Is it supposed to skip this regularization steps from googlelenet？

**Comment by kentsommer** [comment]
It has been a while since I have looked at this. However, I seem to remember that dropout did not significantly improve results from fine-tuning. The original caffe implementation uses a dropout of `0.7`.
